test/common/markdown/markdown.tests.ps1

```diff
@@ -70,7 +67,6 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
                 './docs/testing-guidelines/testing-guidelines.md'
                 './test/powershell/README.md'
                 './tools/*.md'
-                './github/CONTRIBUTING.md'
```

Comment:
(This was a duplicate for anyone wondering, it is the first item on the list)

Reply:
There is other md files. Maybe we need to add
.github/*.md
.github/ISSUE_TEMPLATE/*.md

Reply:
Done.  Added the files.